library/pod-security-policy/users/example.yaml

```diff
@@ -10,3 +10,6 @@ spec:
       image: nginx
       securityContext:
         runAsUser: 250
+        runAsGroup: 250
+        supplementalGroups: 250
+        fsGroup: 250
```

Comment:
looks like `supplementalGroups` and `fsGroup` is only supported as pod security context but not per container basis

```
error: error validating "example.yaml": error validating data: [ValidationError(Pod.spec.containers[0].securityContext): unknown field "fsGroup" in io.k8s.api.core.v1.SecurityContext, ValidationError(Pod.spec.containers[0].securityContext): unknown field "supplementalGroups" in io.k8s.api.core.v1.SecurityContext]; if you choose to ignore these errors, turn validation off with --validate=false
```

not supposed in containers: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#securitycontext-v1-core
supported in pods: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#podsecuritycontext-v1-core